web/src/ApiButton.tsx

```diff
@@ -287,17 +296,6 @@ async function updateButtonStatus(
   await tiltApiPut("uibuttons", "status", toUpdate)
 }
 
-function setHiddenInputs(
-  uiButton: UIButton,
-  inputValues: { [name: string]: any }
-) {
-  uiButton.spec?.inputs?.forEach((i) => {
-    if (i.hidden && i.name) {
-      inputValues[i.name] = i.hidden.value
-    }
-  })
-}
-
```

Comment:
this is unrelated to this PR - I just noticed it was made unused by a previous PR and didn't seem worth its own PR